evm/src/executor/mod.rs

```diff
@@ -148,7 +148,7 @@ impl Default for RawCallResult {
             labels: BTreeMap::new(),
             traces: None,
             debug: None,
-            state_changeset: StateChangeset::new(),
+            state_changeset: None,
```

Comment:
I recall this was an option before, any reason for this change vs empty map?

Reply:
I suppose we could do an empty map instead

Reply:
Yep seems cleaner